poc/.gitignore

```diff
@@ -0,0 +1 @@
+secrets.json
```

Comment:
Endline missing!!

Reply:
Eh, proof-of-concept folder. We'll be more careful with these in the main code.